### PR TITLE
Add default styles and discussion on style customization

### PR DESCRIPTION
**Pull request by domenic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -393,24 +393,70 @@ get removed from the DOM, and if so should there be cases where it remains?
...
+They are similar to [the default styles for `<dialog>`](https://html.spec.whatwg.org/multipage/rendering.html#flow-content-3),
+which are very minimal.
+
+```css
+std-toast {
+    position: fixed;
+    offset-block-end: 0;
+    offset-inline-end: 0;
+    border: solid;
+    padding: 1em;
```
Comment (jackbsteinberg) [inline README.md:405]: Maybe add `margin: 1em` here? Every library in the study group had their default toast slightly offset from the edge of the screen.

File: README.md [hunk 2]
```diff
@@ -393,24 +393,70 @@ get removed from the DOM, and if so should there be cases where it remains?
...
+std-toast [slot="action"] {
+    color: blue;
+    text-transform: uppercase;
+}
+```
+
+(Note: we could offer an alternate syntax of `std-toast::part(action) { ... }`.
+Discuss in [#20](https://github.com/jackbsteinberg/std-toast/issues/20).)
+
+TODO: link to demos.
```
Comment (jackbsteinberg) [inline README.md:459]: The demos will be hosted here (https://jackbsteinberg.github.io/std-toast/) after review, depending on the review timeline we should maybe just add this in

**Review by jackbsteinberg — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (domenic) [reply README.md:405]: I was unsure. I'll change the position to add a 1em offset.

File: README.md (hunk 2 above)
Comment (domenic) [reply README.md:459]: Let's add in the link as part of #45.

**Pull request merged**